x-pack/plugins/ml/public/embeddables/anomaly_charts/use_anomaly_charts_data.ts

```diff
@@ -58,29 +55,41 @@ export function useAnomalyChartsInputResolver(
 
   useEffect(() => {
     const subscription = combineLatest([
-      getJobsObservable(embeddableInput, anomalyDetectorService, setError),
-      embeddableInput,
+      getJobsObservable(api.jobIds$, anomalyDetectorService, setError),
+      api.maxSeriesToPlot$,
+      timeRange$,
+      api.filters$,
+      api.query$,
```

Comment:
did you try using fetch$ instead?

Reply:
Yes please consider using `fetch$` instead. If you'd like to have the fetch context as reactive variables instead of RXJS subjects you can use the util `useFetchContext` which will be added in https://github.com/elastic/kibana/pull/182094

Reply:
Updated here https://github.com/elastic/kibana/pull/183456/commits/81121a7b00f4688b0a4946c72ce0c6c2a688e0ef